fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -1091,6 +1128,22 @@ LSTATUS __stdcall  RegGetValueFixup(
 #ifdef _DEBUG
         Log("[%d] RegGetValue:\n", RegLocalInstance);
 #endif
+
+        // If subkey is in redirected path, use it to get Registry Value
+		const char* updatedSubKey = ReplaceRegistryQueryPath(&key, SubKey);
```

Comment:
This code block is super tabbed.  un-tab to match indentation.

Reply:
Updated